creating_images/s2i_testing.adoc

```diff
@@ -83,13 +83,14 @@ your image name.
 
 ----
 IMAGE_NAME = openshift/ruby-20-centos7
+CONTAINER_ENGINE := $(shell command -v podman 2> /dev/null | echo docker)
 
 build:
-	docker build -t $(IMAGE_NAME) .
+	${CONTAINER_ENGINE} build -t $(IMAGE_NAME) .
```

Comment:
@rhatdan, please indulge me. Usually we use \<variable\> if it's just a standard "replace this" instance. If that's not clear enough, we use \<#\> tags on the command and a line with "\<#\> explanation" under it to explain.
If the $( ) notation doesn't mean anything specific in the makefile context, will you consider updating this command?

Reply:
The ${CONTAINER_ENGINE} here is a real command in Makefile. 
This is basically an environment variable expansion.  The code changes here, will basically run podman if it exists, and then fall back to docker.  Of course it will fail if both don't exists it will fail.

Reply:
BTW This is actual code we use to run some of our test suites in containers on different platforms.

Reply:
@rhatdan, thank you for the clarification!